index.html

```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Tab Butler - a browser tab manager at your service</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
Hey why are these needed?

Reply:
These comes with the font.
<img width="319" alt="Screenshot 2022-07-11 at 8 01 19 AM" src="https://user-images.githubusercontent.com/25580776/178198653-2d4716fb-4add-49be-9804-0ccff0d2229f.png">

Reply:
Removed the old index.html.